ipatests/test_webui/ui_driver.py

```diff
@@ -732,8 +732,15 @@ def button_click(self, name, parent=None,
     def _button_click(self, selector, parent, name=''):
         btn = self.find(selector, By.CSS_SELECTOR, parent, strict=True)
         self.move_to_element_in_page(btn)
```

Comment:
I see an issue here: `scrollIntoView` does not have any effect when called for hidden element. So this line should be moved below WebDriverWait invocation

Reply:
Fixed, waiting for test results...